chaoslib/litmus/disk_fill_by_litmus.yml

```diff
@@ -0,0 +1,161 @@
+- name: Get a Random Pod name
+  shell: >
+    kubectl get pods -l {{ label }} -n {{ namespace }} --no-headers -o custom-columns=:metadata.name | shuf -n 1 
+  args:
+    executable: /bin/bash
+  register: pod_name
+
+- name: Get ephemeral-storage Limit Value
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ namespace }} -o jsonpath='{.spec.containers[?(@.name=="{{ app_container }}")].resources.limits.ephemeral-storage}' | grep -Eo '+|[0-9]+'
+  args:
+    executable: /bin/bash
+  register: limit_value
+
+- name: Get ephemeral-storage Limit storage unit
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ namespace }} -o jsonpath='{.spec.containers[?(@.name=="{{ app_container }}")].resources.limits.ephemeral-storage}' | grep -Eo '[[:alpha:]]+'
+  args:
+    executable: /bin/bash
+  register: limit_value_storage
```

Comment:
How about processing the `litmit_value.stdout` to derive `value` & `unit` without resorting to running it twice with different regex filters. Are ansible's python/jinja based filters inadequate here?